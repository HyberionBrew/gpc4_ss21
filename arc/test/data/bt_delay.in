1: d = 1
2: d = 2
2: r = ()
3: d = 4
4: d = 1
6: d = 2
7: d = 3
7: r = ()
8: r = ()