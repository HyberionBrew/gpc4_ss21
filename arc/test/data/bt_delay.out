4: y = ()
5: y = ()